### ISSUE TITLE
RFE: please include wiki page Introduction.wiki from upstream

### ISSUE THREAD

**Issue by pcfe** [Tetz95/linux-g13-driver#20]
tl;dr: can you please also add the content of https://code.google.com/archive/p/linux-g13-driver/wikis/Introduction.wiki to this repo?

long form: Thanks for forking from deprecated Google Code. Of all the forks on github, yours seems to be the one with the most recent activity. Hence filing an issue here.

When I grabbed [Linux-G13_1.0-r44.zip upstream](https://code.google.com/archive/p/linux-g13-driver/downloads) (before discovering your repo), I initially overlooked the upstream wiki tab but would have preferred to read the `Introduction.wiki` page pretty much right after `README.md`. I think it would be useful to archive that one wiki page into the docs/ directory of this repo.

**Comment by pcfe** [comment]
I think I got the table and the heading depth right, but do check before adding to your repo.
[Introduction.md](https://github.com/Tetz95/linux-g13-driver/files/13601937/Introduction.md)
